lib/senders/sms.js

```diff
@@ -69,6 +81,50 @@ module.exports = (log, translator, templates, config) => {
     }
   }
 
+  function pollCurrentSpend () {
+    let limit
+
+    sns.getSMSAttributes({ attributes: [ 'MonthlySpendLimit' ] }).promise()
+      .then(result => {
+        limit = parseFloat(result.MonthlySpendLimit)
+        if (isNaN(limit)) {
+          throw new Error(`Invalid MonthlySpendLimit "${result.MonthlySpendLimit}"`)
+        }
+
+        const now = new Date()
+        return cloudwatch.getMetricsStatistics({
+          Namespace: 'AWS/SNS',
+          MetricName: 'SMSMonthToDateSpentUSD',
+          StartTime: startOfMonth(now).toISOString(),
+          EndTime: now.toISOString(),
+          Period: SECONDS_PER_WEEK,
+          Statistics: [ 'Sum' ]
+        }).promise()
+      })
+      .then(result => {
+        const { Datapoints: data } = result
+
+        if (! Array.isArray(data)) {
+          throw new Error('Invalid Datapoints')
+        }
+
+        const current = data.reduce((sum, datum) => sum + parseFloat(datum.Sum), 0)
```

Comment:
According to the docs we don't actually need to `parseFloat` here. But since we do need to do it for `MonthlySpendLimit` above, I opted to be consistent (and super-defensive).